content/en/network_monitoring/devices/troubleshooting.md

```diff
@@ -12,91 +12,165 @@ further_reading:
+   **Solution**: 
+   Ensure the following configuration parameters are correct: `user`, `authKey`, `authProtocol`, `privKey`, `privProtocol`.
 
-If you send a feature request, Datadog support needs a `snmpwalk` from the requested device. Run the following and send the output:
+### Traps not being received for devices
 
-```
-snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6
-```
+1. Check the Datadog `agent.log` file to ensure that you aren't unable to bind to the traps port. If this is the case, the following error is present:
 
-#### Why am I only seeing one metric collected for my networks and it's the number of devices collected at zero?
+   ```
+   Failed to start snmp-traps server: error happened when listening for SNMP Traps: listen udp 0.0.0.0:162: bind: permission denied
+   ```
 
-1. Try loosening ACLs/firewall rules for your devices.
-2. Run `snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6` from the host your Agent is running on. If you get a timeout without any response, there is likely something blocking the Datadog Agent from collecting metrics from your device.
+   **Solution**:
+   Add a net bind capability to the Agent binary, which allows the Agent to bind to reserved ports: 
 
-#### What do I do if Datadog supports a vendor or device type but my specific model isn't supported? 
+   ```
+   sudo setcap 'cap_net_bind_service=+ep' /opt/datadog-agent/bin/agent/agent
+   ```
 
-- Contact [Datadog support][1] to put in a request to support your specific model.
-- Extend your profiles to support additional `sysobjectid` values. 
-    For example, if you want to monitor another type of Cisco CSR, you can modify the ISR profile directly to list another `sysobjectid` like this: 
-    
-    ```
-		snmpwalk -v 2c -c [community string] [ip address] 1.3.6.1.2.1.1.2
-    ```
+2. Navigate to the troubleshooting dashboard in NDM:
+
+   {{< img src="/network_device_monitoring/troubleshooting/ndm_troubleshooting_dashboard.png" alt="The Network Device Monitoring page showing the Dashboard dropdown with the NDM Troubleshooting dashboard highlighed." style="width:80%;" >}}
 
-**Note**: If you do not know the `sysobjectid` of your device, try doing an Internet search or run a `snmpwalk` on a host that can reach out to your device. Use the output to list the profile to match against. 
+3. Scroll down to the Traps widget and observe the **Traps incorrectly formatted** graph. If this is non-zero it likely means that the authentication on the NDM collector and the device do not match. 
 
-#### How do I extract information about devices and interfaces of my network devices?
+   {{< img src="/network_device_monitoring/troubleshooting/ndm_traps_dashboard.png" alt="The NDM troubleshooting dashboard showing the Traps widget section." style="width:100%;" >}}
+   
+   **Solution**: 
```

Comment:
the problem/solution flow here just seems unintuitive

Reply:
Yes you're right! I reworked this.